### ISSUE TITLE
Tax rate csv import won't add zip or city when Jetpack is active

### ISSUE THREAD

**Issue by lindamork** [woocommerce/woocommerce#22743]
**Describe the bug**
When Jetpack is active, importing a CSV file that includes zips and cities into Standard Tax Rates will remove the zip and cities from those columns. After import, only wildcards are displayed in the zip/postal code and city columns.

After deactivating Jetpack, I was able to import the CSV and the zip and cities displayed as expected in Standard Rates. I reactivated and reconnected Jetpack after the successful import, and the rates remained with zip codes and cities intact.

**To Reproduce**
1. Make sure Jetpack is activated in Plugins and connected. 
2. Go to 'WooCommerce > Settings > Tax > Standard Rates'
3. Click on 'Import CSV' Here's my CSV import example:  [https://cld.wthms.co/ZQjDMN](https://cld.wthms.co/ZQjDMN)
4. After import, view Standard Tax Rates and notice none of the zip or cities included with the CSV import file are included, only wildcards are showing in those columns.

**Screenshots**
Here's a screenshot of my test CSV file which includes zip codes and cities

![screen shot on 2019-02-14 at 09-35-47](https://user-images.githubusercontent.com/16885584/52806199-3d3c2180-303d-11e9-80e8-0d6177378dd3.png)

With all plugins deactivated except WooCommerce and Jetpack, I import my test CSV and you can see there are no zip codes or cities, just wildcards

![screen shot on 2019-02-14 at 09-35-18](https://user-images.githubusercontent.com/16885584/52806272-62c92b00-303d-11e9-905a-e6444a127c58.png)

After deactivating Jetpack and importing the same CSV again, the zip codes and cities display

![screen shot on 2019-02-14 at 09-40-41](https://user-images.githubusercontent.com/16885584/52806315-855b4400-303d-11e9-9d88-c119da78ca85.png)

**Expected behavior**
I expected the zip codes and cities included in the CSV import file to be displayed in the corresponding columns in the Standard Tax Rates.

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce and Jetpack plugins are active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above. Another HE tested and confirmed.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by kloon** [comment]
@lindamork I tried testing it but not getting the same results, I suspect it might be due to a specific module in Jetpack which is not available when in development mode. Could you perhaps do some tests for me and disable all the Jetpack modules and then enable them one by one testing the import inbetween to find out what module might be the cause? I have a suspicion it might be either the Security module or the Spam one, but have some doubts about the Spam one since we do not call any spam function checks on the data.

I also think this might be something that needs to be addressed in Jetpack rather than WooCommerce since we do not call any Jetpack specific functionality when importing files and processing the data, but let's first determine what is the cause.

**Comment by lindamork** [comment]
@kloon I tested this out with all Jetpack modules disabled, with the same results - no zip or city data added when importing CSV file to standard tax rates, just replaced with wildcards. I got the same results with WordPress 4.9.9 and after updating to 5.0.3. Both running WooCommerce 3.5.4 with Twenty Seventeen or Storefront. I also tested with WooCommerce Services activated (but disabled automated taxes) with the same result. The only way I can successfully import zip and city data is with Jetpack deactivated.

 @LaurenaRehbein kindly agreed to test as well, with similar results. Jetpack development mode might be the key here, as neither of the test sites used are local.
